client/state/data-layer/wpcom/read/streams/index.js

```diff
@@ -242,8 +249,10 @@ const streamApis = {
 				age_based_decay: 0.5,
 				// Default order is by date (latest) unless we're on the recommended tab which shows popular instead.
 				orderBy: streamKeySuffix( streamKey ).includes( 'recommended' ) ? 'popular' : 'date',
-			} ),
-		apiNamespace: 'wpcom/v2',
+			} );
+		},
+		apiNamespace: ( { streamKey } ) =>
```

Comment:
It seems strange that freshly pressed has a different api namespace. Is that just how the endpoint was built? What do you think of adding a comment with the full api path for future reference?

Reply:
Linear issue:
https://linear.app/a8c/issue/CML-785/freshly-pressed-update-freshly-pressed-to-readfreshly-pressed

Reply:
Sounds good, thanks!